test-scripts/local-is-deployment/DeploymentPatterns/pattern-2/infra-provision.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# This script is responsible for provisioning the infrastructure on AWS
+
+echo "This is a Dummy Infrastructure provisioning scripts, Your local machine will be used as Infrastructure"
+echo "Start executing the deployment script"
+CURRENT_PATH=$(pwd)
+bash $CURRENT_PATH/DeploymentPatterns/pattern-2/deploy.sh
```

Comment:
Why do you call the deploy.sh here? It should be called TestGrid not infra-provision.sh.

Reply:
We do not support multiple scripts from the Testgrid side, Hence added this as a temporary workaround. @kasunbg How should we proceed?